src/main/java/codeu/controller/AdminServlet.java

```diff
@@ -88,16 +101,46 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
     String username = (String) request.getSession().getAttribute("user");
     User user = userStore.getUser(username);
 
-    int userCount = userStore.Count();
-    int messageCount = messageStore.Count();
-    int conversationCount = conversationStore.Count();
-
     if (user != null && user.getIsAdmin()) {
```

Comment:
I think I wrote about this the last time. I'm not sure why you have to check if the user is null. Wouldn't you just need to see if they are an admin or not?

Reply:
If I try to call `user.getIsAdmin()` but the user turns out to be null, I'll get a NullPointerException.